app/helpers/model_name_helper.rb

```diff
@@ -0,0 +1,5 @@
+module ModelNameHelper
+  def model_name(record)
+    record.class.model_name.name
```

Comment:
Why we use here `model_name`? Can we change this line a little bit?
```suggestion
    record.class.name
```